Objects/moduleobject.c

```diff
@@ -599,7 +599,7 @@ _PyModule_ClearDict(PyObject *d)
 
     /* First, clear only names starting with a single underscore */
     pos = 0;
-    while (PyDict_Next(d, &pos, &key, &value)) {
+    while (_PyDict_Next(d, &pos, &key, &value, NULL, NULL)) {
```

Comment:
Yes, Same thing. PyDict_Next is slightly more expensive (and also resolves values if passing the pointer). We don't want the values to be resolved here in this case, because they're being cleared.